common/src/main/java/com/sequenceiq/cloudbreak/event/ResourceEvent.java

```diff
@@ -148,6 +148,8 @@ public enum ResourceEvent {
     ENVIRONMENT_UPGRADE_CCM_VALIDATION_FAILED("environment.upgrade.ccm.validation.failed"),
     ENVIRONMENT_UPGRADE_CCM_ON_FREEIPA_STARTED("environment.upgrade.ccm.freeipa.started"),
     ENVIRONMENT_UPGRADE_CCM_ON_FREEIPA_FAILED("environment.upgrade.ccm.freeipa.failed"),
+    ENVIRONMENT_UPGRADE_CCM_TUNNEL_UPDATE_STARTED("environment.upgrade.ccm.tunnelupdate.started"),
+    ENVIRONMENT_UPGRADE_CCM_TUNNEL_UPDATE_FAILED("environment.upgrade.ccm.tunnelupdate.failed"),
```

Comment:
Sanity check: this notification does not seem to be sent anywhere at the moment. Are you planning to add that later when handling CCM upgrade failure recovery?

Reply:
it was missing from UpgradeCcmActions, thanks!